src/languages/swift.js

```diff
@@ -215,22 +219,22 @@ export default function(hljs) {
 
   // https://docs.swift.org/swift-book/ReferenceManual/Attributes.html
   const AVAILABLE_ATTRIBUTE = {
-    begin: /(@|#)available\(/,
-    end: /\)/,
-    keywords: {
-      $pattern: /[@#]?\w+/,
-      keyword: Swift.availabilityKeywords
-        .concat([
-          "@available",
-          "#available"
-        ])
-        .join(' ')
-    },
-    contains: [
-      ...OPERATORS,
-      NUMBER,
-      STRING
-    ]
+    begin: /(@|#)available/,
+    className: "keyword",
+    starts: {
+      contains: [
+        {
+          begin: /\(/,
+          end: /\)/,
+          keywords: Swift.availabilityKeywords.join(' '),
+          contains: [
+            ...OPERATORS,
+            NUMBER,
+            STRING
+          ]
+        }
+      ]
+    }
```

Comment:
I think this is slightly simpler and easier to follow.

Reply:
But this highlights the entire thing (including identifiers and punctuation) as a keyword I think? This is what I want to avoid. Generally only the attribute title itself is highlighted as the keyword. For example:

<img width="566" alt="Screen Shot 2020-12-26 at 11 07 55" src="https://user-images.githubusercontent.com/1496250/103149542-9c573b00-476a-11eb-8603-6e0dd9f1e7a0.png">

As we don't support highlighting identifiers and argument labels, I also didn't support them in `AVAILABLE_ATTRIBUTE`. However, setting a general `className: "keyword"` will highlight the entire thing as a keyword I think?

Reply:
No.  The first mode terminates immediately after `begin` since there is no end... and then `starts` immediately starts a new mode... All tests are green.  Feel free add a failing test if I'm missing something but pretty sure I go tthis one right.

Reply:
I think this question is the only blocker?  So let me know if you find a testable issue here. :)  This missed the 10.5 window but will still be great to get it merged. :)

Reply:
Ah, never mind. I just didn't see the `starts` in there.

Do we still need the `contains` in the middle? There's only one rule after all, and there can only be one occurrence of the argument list.

Reply:
Technically PERHAPS you could get away with just `starts: { end: /\)/, ... }` but I'm not sure it would be as obvious what is happening at a glance so that doesn't predispose me to simplify further. :)